### DISCUSSION TITLE
Homepage Styles

### DISCUSSION THREAD

**Discussion by Confectrician** [openhab/openhab-docs#2055] [Team Posts]
Hi all,

I have seen that there is a `_sass` folder existing, but nearly nothing in there.
Are we even using that?!

I was looking for doing a fix with the strangely viewed footer,
but don't want to break anything now or get the chagnes overwritten, with the next sass run.

**Comment by ThomDietrich** [comment]
No they are not used. (Never payed much attention. I guess we should just delete them.)
Regarding the footer: Another aspect of the page I was hoping to see implemented in *the new webpage design*. Basically this should be solved by the help of HTML5 semantic elements.

The new page design is sadly something no one is currently working on. I would like to push this project further but I'm dedicated to other tasks right now.

With the above said: It would be great if you could improve the current page! Just know that we'll eventually (could take months) replace it.

**Comment by Confectrician** [comment]
I would love to make progress with a new design, but i am really busy with my master degree next months.
So i am facing a lack of time too.

I will think about removing the sass stuff for now.
In general i would appreciate using sass, since i like it.
But porting the currently used stuff doesn't seem very useful, when we are going for a new design anyways on long term.

**Comment by ThomDietrich** [comment]
Exactly ;)
